drone/build/build_stop.go

```diff
@@ -20,13 +20,22 @@ func buildStop(c *cli.Context) (err error) {
 	if err != nil {
 		return err
 	}
+
 	number, err := strconv.Atoi(c.Args().Get(1))
 	if err != nil {
-		return err
+		return errInvalidBuildNumber
 	}
-	job, _ := strconv.Atoi(c.Args().Get(2))
-	if job == 0 {
+
```

Comment:
this could be simplified

```
job, err := strconv.Atoi(c.Args().Get(2)
if job == "" {
  job = 1
} else if err != nil {
  return <job error>
}
```